projects/fsastorefront-e2e-cypress/cypress/helpers/changeRequest.ts

```diff
@@ -1,5 +1,6 @@
 const currentDate = Cypress.moment().format('DD-MM-YYYY');
 const todaysDate = Cypress.moment().format('DD MMM YYYY');
```

Comment:
currentDate & todaysDate are not needed anymore? i dont see that they are used in this file...
Please remove them if they are not needed

Reply:
Removed